src/main/java/appeng/core/sync/network/NetworkHandler.java

```diff
@@ -20,64 +20,76 @@
+		NetworkHandler.INSTANCE.registerMessage( PacketCompassResponse.class, PacketCompassResponse.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketCompressedNBT.class, PacketCompressedNBT.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketConfigButton.class, PacketConfigButton.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketCraftRequest.class, PacketCraftRequest.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketInventoryAction.class, PacketInventoryAction.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketInventoryAction.class, PacketInventoryAction.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketLightning.class, PacketLightning.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketMatterCannon.class, PacketMatterCannon.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketMEInventoryUpdate.class, PacketMEInventoryUpdate.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketMockExplosion.class, PacketMockExplosion.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketMultiPart.class, PacketMultiPart.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketNEIRecipe.class, PacketNEIRecipe.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketNewStorageDimension.class, PacketNewStorageDimension.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketPaintedEntity.class, PacketPaintedEntity.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketPartialItem.class, PacketPartialItem.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketPartPlacement.class, PacketPartPlacement.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketPatternSlot.class, PacketPatternSlot.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketProgressBar.class, PacketProgressBar.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketProgressBar.class, PacketProgressBar.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketSwapSlots.class, PacketSwapSlots.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketSwitchGuis.class, PacketSwitchGuis.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketSwitchGuis.class, PacketSwitchGuis.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketTransitionEffect.class, PacketTransitionEffect.class, disc++, Side.CLIENT );
+		NetworkHandler.INSTANCE.registerMessage( PacketValueConfig.class, PacketValueConfig.class, disc++, Side.SERVER );
+		NetworkHandler.INSTANCE.registerMessage( PacketValueConfig.class, PacketValueConfig.class, disc++, Side.CLIENT );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "disc". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1854)